packages/snaps-jest/src/test-utils/controller.ts

```diff
@@ -24,6 +24,16 @@ export const getRootControllerMessenger = (mocked = true) => {
       'ExecutionService:handleRpcRequest',
       jest.fn(),
     );
+
+    messenger.registerActionHandler(
+      'ApprovalController:hasRequest',
+      () => true,
+    );
+
+    messenger.registerActionHandler(
+      'ApprovalController:acceptRequest',
+      async (_id: string, value: unknown) => Promise.resolve({ value }),
```

Comment:
Since it's an async function, `Promise.resolve` is redundant.
```suggestion
      async (_id: string, value: unknown) => ({ value }),
```